src/DocumentDB.ChangeFeedProcessor.UnitTests/LeaseManagement/DocumentServiceLeaseTests.cs

```diff
@@ -63,6 +66,7 @@ public void ValidateSerialization_AllFields()
             DocumentServiceLease originalLease = new DocumentServiceLease
             {
                 Id = "id",
+                LeasePartitionKey="leasePk",
```

Comment:
let's not modify existing tests but add new tests for the case when lease collection is partitioned by partitionKey.